### DISCUSSION TITLE
Connection to server too unstable or fragile

### DISCUSSION THREAD

**Discussion by Danielgb23** [mm201/pkmnFoundations#88] [Q&A]
I use a DS lite with pokemon platinum and followed the instructions of the secondary DNS but my connection is way to fragile to allow a trade to occur. There is no problem logging on to WFC and reaching the server apparently. But  then at "Checking GTSs status..." it breaks and gives "Communication error...". If my connection is bad does it make that?

**Comment by mm201** [comment]
That sounds like a problem on your end. Are you using a DNS that has PCN support?

**Comment by Danielgb23** [comment]
I used this one DNS: 178.62.43.212
From the page https://pkmnclassic.net/ 
Are there alternatives?

**Comment by mm201** [comment]
It’s possible your ISP is doing something dodgy with DNS or unencrypted HTTP traffic. You could try a different internet connection or a hardware-based VPN. There’s not much I can do on my end to troubleshoot connection problems.

**Comment by ghost** [comment]
Yeah/ that definitely sounds like a problem to me.
